src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiJsonSerializer.cs

```diff
@@ -16,8 +17,8 @@ internal class OpenApiJsonSerializer : FormulaValueSerializer, IDisposable
         private bool _topPropertyWritten = false;
         private bool _wasDisposed;
 
-        public OpenApiJsonSerializer(bool schemaLessBody)
-            : base(schemaLessBody)
+        public OpenApiJsonSerializer(IRuntimeContext context, bool schemaLessBody)
```

Comment:
>IRuntimeContext [](http://example.com/codeflow?start=37&length=15)

Can we use a more minimal interface here?  Ie, runtimeContext has everything (EvalVisitor), what's the minimal knowledge we need just for deserializing?

Reply:
Done